README.md

```diff
@@ -8,5 +8,11 @@ that defines a "best practice". Anecdotes and personal experience have
 value, but we prefer evidence from research, books, and large and mature
 communities of practice.
 
+We have agreed to this practices by consensus. We document _what_ to do and
+_why_, and only recommend optional resources about _how_ to do it. These
+practices are aspirational but when they can't be followed it's nice to explain
+why. They document heuristics, as detailed instructions may never cover all
+scenarios.
+
```

Comment:
@MonikaFu, I elevated this paragraph to README because it applies to all themes. Do you agree?